CONFIG.md

```diff
@@ -355,6 +355,13 @@ This is an example of the settings for the validate tasks:
 
 ```
 {
+  "eslint": {
+    "srcPaths": [
```

Comment:
We've already got some issues like this, it should probably be it's own task to correct.